nakama/lib/src/enum/leaderboard_operator.dart

```diff
@@ -1,7 +0,0 @@
-enum LeaderboardOperator {
```

Comment:
why was this removed?

Reply:
It's been moved to the leaderboard.dart file.